disco_aws_automation/disco_rds.py

```diff
@@ -409,12 +413,19 @@ def modify_db_parameter_group(self, db_parameter_group_name, parameters):
                                                                'MinimumEngineVersion': 'someversion',
                                                                'ApplyMethod': 'pending-reboot'}])
 
-    def recreate_db_parameter_group(self, database_name, db_parameter_group_name,
+    def recreate_db_parameter_group(self, env_name, database_name, db_parameter_group_name,
```

Comment:
But I dont see any value being passed to the function other than `self.vpc_name`. Is there any new code which is not part of this PR?